nikola/plugins/command/check.py

```diff
@@ -46,7 +46,10 @@
 from nikola.utils import get_logger, STDERR_HANDLER
 
 
-def _call_nikola_list(site):
+def _call_nikola_list(site, cache=None):
+    if cache is not None:
+        if 'files' in cache and 'deps' in cache:
```

Comment:
Yes, because this checks if the values are in the cache. (Just checking one of the two is fine as well.)